src/components/header/index.tsx

```diff
@@ -21,6 +21,7 @@ const Header: FC<Props> = ({ authService }) => {
     <InternalHeader
       username={authService.getUser()?.name}
       onLogout={logOutAndRedirect}
+      useDemoLogo={env.ENV === 'development'}
```

Comment:
Bruk gjerne enum'en her 👍 
```suggestion
      useDemoLogo={env.ENV === Environment.DEVELOPMENT
```

Reply:
He he, ja selvfølgelig, tåpelig feil :blush: